checker/jtreg/sortwarnings/ErrorOrders.out

```diff
@@ -1,50 +1,140 @@
-OrderOfCheckers.java:12:99: compiler.err.proc.messager: (assignment.type.incompatible)
```

Comment:
I needed to expand this file, otherwise the first for messages would be reduced to a single one.